### ISSUE TITLE
Setting breakpoints on Gatsby-plugin-image <staticImage> fails

### ISSUE THREAD

**Issue by LarsEjaas** [gatsbyjs/gatsby#30603]
I am trying to set breakpoints on a <StaticImage> in fullWidth layout.

As per documentation:

breakpoints | [750, 1080, 1366, 1920] | Output widths to generate for full width images. Default is to generate widths for common device resolutions. It will never generate an image larger than the source image. The browser will automatically choose the most appropriate.
-- | -- | --

<StaticImage> components will accept settings as props
 so I would expect it to be done like this:
 
```
 <StaticImage
              Tag="section"
              alt="image Alt"
              src="../image.png"
              placeholder="blurred"
              layout="fullWidth"
              breakpoints={[325, 512, 699]}
            />
```

... but I have tried combining different thinks like strings numbers arrays and objects without any luck?

**Comment by dev-szymon** [comment]
issue already reported #30353 
pr with fix #30451

**Comment by LarsEjaas** [comment]
Looks like a fix is already on the way! Perfect 👌
